README.md

```diff
@@ -1 +1,67 @@
-# webpack-form
\ No newline at end of file
+# Webpack - Form
+
+> This is a project to create and a to do list with js using webpack!
+
+
+![screenshot](./assets/img/screenshot.png)
+
+## Built With
+
+- HTML + CSS + Javascript
+
+## Getting Started
+
+* You will not need additional instructions on installing Prerequisites.
+* In case you have node_modules on your folder Add them to .gitignore file before you a push.
+
+### Prerequisites
+
+* No prerequisites needed for this project.
+ <!--
+This is an example of how to list things you need to use the software and how to install them.
+* npm
+  ```sh
+  npm install npm@latest -g
+  ```
+-->
+
+### Installation
+<!-- 1. Get a free API Key at [https://example.com](https://example.com) -->
+1. Clone the repo
+   ```sh
+   git clone git@github.com:smunozmo/awesome-books.git
+   ```
+2. Install NPM packages
+   ```sh
+   npm install
+   ```
+<!-- 4. Enter your API in `config.js`
+   ```JS
+   const API_KEY = 'ENTER YOUR API';
+   ```
+-->
```

Comment:
* Kindly add more information on how to run this project locally this would make it easier for anyone who wants to clone and run your project locally

Reply:
fixed